content/blog/2022-03-02-out-in-the-open-februar-2022.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Zwischen Fahrradsensoren und Solardachkataster – Out in the Open Februar 2022"
+author: "Tim Fangmeyer & Knut Hühne - OK Lab Berlin, Klara Juhl - OK Lab Osnabrück"
+date: '2022-03-02T09:45:00+01:00'
+excerpt: Monatlich erscheinender Überblick über Entwicklungen in der Open Data und Civic Tech Szene
+topic: community
+images:
+- imgname: out-in-the-open-2022-02-header.jpg
+  attribution: Foto von [Eriel Suarez](https://unsplash.com/@erielsuarez) auf [Unsplash](https://unsplash.com/photos/Rthm2XMBi4s)
+og_image: out-in-the-open-2022-02-header.jpg
+type: blog
+---
+Der Landtag Schleswig-Holstein hat am 23. Februar ein neues [Digitalisierungsgesetz beschlossen](https://www.schleswig-holstein.de/DE/Landesregierung/V/_startseite/Artikel2022/220221_digitalisierungsgesetz.html). Mit dem neuen Gesetz soll es zum Standard werden, wichtige Behördengänge komplett online erledigen zu können. Zudem sollen noch mehr Informationen der Landesverwaltung im landeseigenen Open-Data-Portal veröffentlicht werden. Ob das Gesetz tatsächlich Vorbildcharakter aufweist, offene Daten zum Standard zu machen? Das wird sich in den nächsten Monaten zeigen.
+
+Aus Sachsen-Anhalt kommt in diesem Monat ein Beispiel für den Nutzen offener Geodaten: Holger Naumann ([Regionale Planungsgemeinschaft Anhalt-Bitterfeld-Wittenberg](https://www.planungsregion-abw.de/)) hat frei verfügbare Oberflächenmodelle für Schattenberechnungen verwendet. In Kombination mit Angaben zu Dachformen und Dachausrichtungen hat er im Anschluss die Beta-Version eines Solarpotenzial-Katasters entwickelt, [welche bereits öffentlich abrufbar ist](https://ris.planungsregion-abw.de/mapbender/application/pv_dachflaechenpot_rpg_abw). 
+
+Das Badische Landesmuseum wird 10.000 Objekte aus der hauseigenen Sammlung [unter CC0-Lizenz stellen](https://twitter.com/BLM_Karlsruhe/status/1493978681898897413?t=rAOzYC6MqhhL-kxnVxgZaw&s=19). Unter die CC0-Lizenz gestellte Inhalte können ohne Nachfrage kopiert, veröffentlicht oder beliebig verwendet werden. Die Objekte sollen im digitalen Katalog des Museums für alle Nutzer:innen zur Verfügung gestellt werden. „Unsere Sammlungen gehören den Bürgerinnen und Bürgern. [Das] ist ein weiterer Schritt der Teilhabe", so der Direktor des Landesmuseums, Prof. Dr. Eckart Köhne.
+
+Auf Twitter entspann sich in der Open-Data-Community [ein kleines Gedankenspiel](https://twitter.com/MisterOpenData/status/1495731283539935246?t=UKDvMjZbXoo6ENgRjlYEZg&s=19), wie man die Veröffentlichung von Offenen Daten mit einer einmaligen Geldsumme am erfolgreichsten fördern könne. "Stellt euch vor, ihr hättet 1 Million € für #opendata zur Verfügung. Was würdet ihr damit sinnvoll anfangen?", fragt Twitter-User @MisterOpenData. Die Antworten reichen von der Verbesserung von Metadaten bis hin zum Einladen von Programmierer:innen in die Verwaltung, um direkt vor Ort gemeinsam technische Lösungen zu entwickeln und abzustimmen.
```

Comment:
Eher: In einem Die Open-Data Community hat vor kurzem ein kleines theoretisches Szenario durchgespielt..